capstone_wrapper.cpp

```diff
@@ -513,13 +513,13 @@ bool Capstone::IsBranchGoingToExecute(size_t cflags, size_t ccx) const
     case X86_INS_JE:
         return bZF;
     case X86_INS_JGE:
-        return bSF == bOF;
+        return bSF == bOF || bZF;

```

Comment:
Refer to the official manual:

![jge](http://i.prntscr.com/3de21a2caec742eb96ab1d992331117f.png)

http://www.intel.com/content/dam/www/public/us/en/documents/manuals/64-ia-32-architectures-software-developer-instruction-set-reference-manual-325383.pdf